openslides_backend/action/actions/motion_submitter/create.py

```diff
@@ -53,4 +57,14 @@ def update_instance(self, instance: Dict[str, Any]) -> Dict[str, Any]:
         exists = self.datastore.exists(collection=self.model.collection, filter=filter)
         if exists:
             raise ActionException("(user_id, motion_id) must be unique.")
+        if instance.get("weight", 10000) == 10000:
+            self.set_weight(instance)
```

Comment:
You can just remove the default value from the models file, since you will always either set one automatically or use the provided one

Reply:
Okay, updated the model and use a different check here.